test/cli/scenarios_to_cli/other /scenario_validate_volume_whitelist/policy.yaml

```diff
@@ -7,9 +7,10 @@ spec:
   rules:
   - name: validate-volumes-whitelist
     match:
-      resources:
-        kinds:
-        - Pod
+      any:
+        - resources:
+              kinds:
+                - Pod
     validate:
       message: "Volumes white list"
```

Comment:
I know you didn't do this, but please change this message to something useful like "Volume type is not of type hostPath, emptyDir, or configMap."